clients/cloud/rust/src/consumer.rs

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+extern crate futures;
+extern crate rdkafka;
+extern crate tokio;
```

Comment:
It might be worth considering using Rust 2018 and removing these `extern crate` statements.

Reply:
Nice! Didn't know these weren't required in Rust 2018 :)